polaris-intellisense/src/extension.ts

```diff
@@ -0,0 +1,35 @@
+// The module 'vscode' contains the VS Code extensibility API
```

Comment:
**suggestion/nit (non-blocking):** Can we tidy up the double newlines throughout the file?